client/app/telecom/sms/options/response/telecom-sms-options-response.html

```diff
@@ -175,99 +175,41 @@ <h1 data-translate="sms_options_response"></h1>
-                                                <button
-                                                    type="button"
-                                                    class="btn btn-link"
-                                                    data-ng-click="TelecomSmsOptionsResponseCtrl.removeTrackingOptions($index, option)"
-                                                    data-translate="delete">
-                                                </button>
-                                            </li>
-                                        </ul>
-                                    </div>
-                                </td>
-                            </tr>
-                        </tbody>
-                        <tbody data-ng-if="TelecomSmsOptionsResponseCtrl.loading.init">
-                            <tr>
-                                <td colspan="3">
-                                    <oui-spinner></oui-spinner>
-                                </td>
-                            </tr>
-                        </tbody>
-                        <tbody data-ng-if="!TelecomSmsOptionsResponseCtrl.loading.init && !TelecomSmsOptionsResponseCtrl.smsResponse.trackingOptions.length">
-                            <tr>
-                                <td class="text-center"
-                                    colspan="3"
-                                    data-translate="sms_options_response_tracking_empty">
-                                </td>
-                            </tr>
-                        </tbody>
-                    </table>
-                </div>
+                <oui-datagrid class="reduce-padding" rows="TelecomSmsOptionsResponseCtrl.smsResponse.trackingOptions">
+                    <oui-column title="'sms_options_response_tracking_media_label' | translate" property="media" sortable>
+                        <span class="text-capitalize visible-xs"
+                              data-ng-bind="$row.media">
+                        </span>
+                        <i class="ovh-font ovh-font-1x4 ml-2 hidden-xs"
+                            title="{{ $row.media }}"
+                            data-ng-class="{
+                                'ovh-font-mail': $row.media === 'email',
+                                'ovh-font-message': $row.media === 'sms'
+                            }">
+                        </i>
+                    </oui-column>
+
+                    <oui-column property="target">
+                        <strong class="d-block"
+                                data-ng-bind="$row.target">
+                        </strong>
+                        <span data-translate="sms_options_response_tracking_email_sender_label"
+                            data-translate-values="{ sender: $row.sender }">
+                        </span>
+                    </oui-column>
+
+                    <oui-action-menu align="end" compact>
```

Comment:
A recent discussion about `data-` prefixes lead us to keep all `data-` on custom attributes whenever we are in the managers. => `data-align`, `data-compact`, same thing for the datagrid attributes especially for `title` but except for `id`, `name`, etc...